css/fonts.css

```diff
@@ -12,26 +6,33 @@ h1, h2, h3{
 
 
 h1{
```

Comment:
changed font size + colour for my main heading; this is for the login page title